blogposts/structural-code-search-at-scale-from-your-browser.md

```diff
@@ -0,0 +1,305 @@
+---
+title: Structural code search. At scale and from your browser.
+author: Rijnard van Tonder
+authorUrl: https://twitter.com/rvtond
+publishDate: 2020-01-24T10:00-07:00
+tags: [blog]
+slug: structural-code-search-at-scale-from-your-browser
+heroImage: https://TODO
+published: true
+---
+
+We're introducing a richer way to search code at Sourcegraph with structural
+code search. And we're making it available at scale.
+
```

Comment:
Adding a section about why literal and reg expression searches with an example will help clarify why structural search is needed before jumping into what it is.